pkg/model/common.proto

```diff
@@ -30,7 +30,7 @@ enum ApplicationKind {
 message ApplicationGitPath {
     // The repository that was configured at piped.
     ApplicationGitRepository repo = 1 [(validate.rules).message.required = true];
-    string path = 2 [(validate.rules).string.min_len = 1];
+    string path = 2 [(validate.rules).string.pattern = "^[^/].+$"];
```

Comment:
nits, in case the path contains one character, `"^[^/].*$"` would be better 👀
( looks like we support path's min_len = 1 previously )